FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/FhirApiDataClient.cs

```diff
@@ -89,7 +76,7 @@ public sealed class FhirApiDataClient : IFhirDataClient
                 response.EnsureSuccessStatusCode();
                 _logger.LogInformation("Successfully retrieved search result for url: '{url}'.", searchUri);
 
-                bundleContent = await response.Content.ReadAsStringAsync(cancellationToken);
+                return await response.Content.ReadAsStringAsync(cancellationToken);
```

Comment:
Will ```fhirBundleResult``` certain be a ```bundle``` if ```statusCode``` is success? Just curious why we had parseParseExcetion in previous deserialization but now don't.

Reply:
yes, it should be a bundle json. At least should be a json in content if we successfully retrieved the body. When deserializing from JObject, we will return empty when the required field is missing.

Reply:
Can we add ```logWarning``` somewhere to alert empty resources be extracted from searched string?

Reply:
That's a good advice. Added error handling when the queried json is not correct